job-simulation.rb

```diff
@@ -2,3 +2,32 @@
 require './Queue.rb'
 
 
+ class stupid_job_fire_method
+   attr_accessor :waiting, :employees
+
+  def initialize
+    @employees = Stack.new()
+     (0..5).each do |number|
+       @employees.push(number)
+     end
+    @waiting = Queue.new()
+     (6..10).each do |number|
+       @waiting.enqueue(number)
+      end
+  end
```

Comment:
This looks good good. The only suggestion I have is that making this more adaptable for possible changes to the numbers of workers/waiting.